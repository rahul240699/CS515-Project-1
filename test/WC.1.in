python prog/wc.py std_in.txt