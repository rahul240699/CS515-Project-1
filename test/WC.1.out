5     15    72    test/WC.1.in